readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -68,86 +69,112 @@ function attach_elastic_search_query(data) {
 
                             var contents = $('<div class="context">');
 
+                            var section_template = '' +
+                                '<div>' +
+                                    '<a href="<%= section_subtitle_link %>">' +
+                                        '<%= section_subtitle %>' +
+                                    '</a>' +
+                                '</div>' +
+                                '<% for (var i = 0; i < section_content.length; ++i) { %>' +
+                                    '<div>' +
+                                        '<%= section_content[i] %>' +
+                                    '</div>' +
+                                '<% } %>';
```

Comment:
These strings are really cumbersome, is there not a good way to do multi-line strings in JS? :(

Reply:
There are, but eslint is not allowing any: https://travis-ci.org/readthedocs/readthedocs.org/jobs/556276376#L593

Reply:
@davidfischer thoughts here? Is it too fancy and new?